src/guest_webapp/Dockerfile

```diff
@@ -1,19 +1,21 @@
-# choose node
 FROM node:20-alpine AS build_react
 
-# set working directory inside the container
-WORKDIR /guest_webapp
+WORKDIR /app
 
-# copy package.json & package-lock.json
 COPY package*.json ./
-# install dependencies/packages
+
 RUN npm install
 
-# copy all the project files
 COPY . .
 
-# react default expose port
-EXPOSE 5173
+RUN npm run build
+
+FROM nginx:alpine
+
+COPY nginx.conf /etc/nginx/conf.d/default.conf
```

Comment:
NO ACTION REQUIRED

Copy uses relative pathing, so if you want, you can do stuff like
```
../../docker-compose/configs/spa-nginx.conf
```

That would let you have a shared config file. 

In this case, guest&staff will eventually need unique configs, so keep them separate! 👍